### PR TITLE
GCI Task : JUnit Tests added

### PR DESCRIPTION
**Pull request by w0lF07**
**testEquals()** checks the value of answers against given value
**testTrue()** checks for the correct answer
**testFalse()** checks for incorrect answer
**testNull()** checks for void values
**sampleTest()** prints a simple output
**testArrayEquals()** matches the values

### CODE REVIEW

File: src/test/java/org/jbossoutreach/w0LF07_JUnit_tests [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package org.jbossoutreach;
```
Comment (shubhamvashisht) [inline src/test/java/org/jbossoutreach/w0LF07_JUnit_tests]: Change file extension to .java

**Comment by w0lF07** [comment]
Is it correct now @shubhamvashisht

**Comment by w0lF07** [comment]
@shubhamvashisht @opticod is it done now???

File: src/test/java/org/jbossoutreach/w0LF07.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
+package org.jbossoutreach;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class w0LF07
```
Comment (opticod) [inline src/test/java/org/jbossoutreach/w0LF07.java]: java class names should start with uppercase letter.

File: src/test/java/org/jbossoutreach/w0LF07.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+{
+
+    private int a = 10;
+    private int b = 7;
+    private String s1 = "SWASTIK";
+    private String s2 = "DAS";
+    private String Null1 = null;
```
Comment (opticod) [inline src/test/java/org/jbossoutreach/w0LF07.java]: Java variable name should start with lowercase letter.

File: src/test/java/org/jbossoutreach/w0LF07.java (hunk 2 above)
Comment (opticod) [reply src/test/java/org/jbossoutreach/w0LF07.java]: Don't forget to change the file name also.

**Comment by opticod** [comment]
Follow this for more naming convention guidelines https://www.javatpoint.com/java-naming-conventions

**Comment by w0lF07** [comment]
@shubhamvashisht  @opticod  is it done now???

**Pull request merged**